lotus/client.go

```diff
@@ -242,16 +242,10 @@ func monitorLotusSync(ctx context.Context, c *API) {
 }
 
 func refreshHeightMetric(ctx context.Context, c *API) {
-	var h uint64
-	state, err := c.SyncState(ctx)
+	heaviest, err := c.ChainHead(ctx)
```

Comment:
Simplify `curent height` metric in grafana dashboard to the current heaviest tipset... not the newer _potential_ new tipset that we discovered recently.